pkgs/by-name/ap/apfsprogs/package.nix

```diff
@@ -1,29 +1,32 @@
-{ lib
-, stdenv
-, fetchFromGitHub
-, nixosTests
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  nixosTests,
 }:
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "apfsprogs";
-  version = "0-unstable-2024-09-27";
+  version = "0.2.0";
 
   src = fetchFromGitHub {
     owner = "linux-apfs";
     repo = "apfsprogs";
-    rev = "f31d7c2d69d212ce381399d2bb1e91410f592484";
-    hash = "sha256-+c+wU52XKNOTxSpSrkrNWoGEYw6Zo4CGEOyKMvkXEa0=";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-rolbBLdE98jqlKC06fTo6eJU3abKzgB3QIlaw4bza9U=";
   };
 
-  postPatch = let
-    shortRev = builtins.substring 0 9 finalAttrs.src.rev;
-  in ''
-    substituteInPlace \
-      apfs-snap/Makefile apfsck/Makefile mkapfs/Makefile apfs-label/Makefile \
-      --replace-fail \
-        '$(shell git describe --always HEAD | tail -c 9)' \
-        '${shortRev}'
-  '';
+  postPatch =
+    let
+      shortRev = builtins.substring 0 9 finalAttrs.src.rev;
```

Comment:
This no longer makes sense, the src.rev is a tag.

Reply:
Good catch! Thanks for noticing.